basicObstacleAvoidance.ino

```diff
@@ -18,7 +18,7 @@
 
         void loop() {
 
-            if(front.getDistance() <= 15) {
+            if(front.getDistance() <= 15 && front.getDistance != 0) {
```

Comment:
OK, the first problem is that it does not compile, there's still a small mistake.
Aside of this, there are *two* calls to `front.getDistance` which may yield different results. For example, what would happen if the first call would return `0` (e.g. no obstacle is found) and the second one returns `60`? Would that be the correct behavior?